src/core/IPstructs.h

```diff
@@ -12,6 +12,7 @@ namespace core {
     {
         std::shared_ptr<IPaddressBase> Ip;
         std::shared_ptr<IPmaskBase> NetMask;
+        QString SubName = "blank";
```

Comment:
Jeżeli zmienną nazywasz SubName to umieściłbym ją w Subnet, bo dla sieci głównej nie za bardzo to ma sens.